src/config/config.ts

```diff
@@ -31,21 +31,21 @@ const env = envalid.cleanEnv(process.env, {
     CANDIDATE_TTL_SEC: envalid.num({ default: 120 }),
     SESSION_TTL_SEC: envalid.num({ default: 86400 }),
     ASAP_PUB_KEY_TTL: envalid.num({ default: 3600 }),
-    SYSTEM_ASAP_BASE_URL_MAPPINGS: envalid.json({ example: '[{"kid": "kidPattern", baseUrl": "https://asap.org"}]' }),
-    SYSTEM_ASAP_JWT_AUD: envalid.str(),
-    SYSTEM_ASAP_JWT_ACCEPTED_HOOK_ISS: envalid.str(),
-    JITSI_ASAP_BASE_URL_MAPPINGS: envalid.json({ example: '[{"kid":"kidPattern","baseUrl":"https://asap.org"]' }),
-    JITSI_ASAP_JWT_AUD: envalid.str(),
-    JITSI_ASAP_JWT_ACCEPTED_HOOK_ISS: envalid.str(),
+    SYSTEM_ASAP_BASE_URL_MAPPINGS: envalid.json({ default: '[{"kid": "^jitsi/(.*)$", "baseUrl": "https://localhost:18443/server"}]' }),
```

Comment:
```suggestion
    SYSTEM_ASAP_BASE_URL_MAPPINGS: envalid.json({
        default: [ { 'kid': '^jitsi/(.*)$',
            'baseUrl': 'https://localhost:18443/server' } ]
     }),
```
Default needs to be an object, not a string